README.md

```diff
@@ -2,15 +2,47 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Testing in each PR
+## Cointinuous Deployment and commit format
+
+We use [semantic release](https://github.com/semantic-release/semantic-release) to generate each tag for automatic versioning, that's why it's important to have each commit formatted correctly, this tool uses [Angular commit message](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format) format by default.
+The format is the following:
```

Comment:
Maybe you can add a link to [Conventional Commits Specification](https://www.conventionalcommits.org/en/v1.0.0/)